tests/unit/sequential/test_par.py

```diff
@@ -935,3 +931,18 @@ def test_load(self, mock_file, cloudpickle_mock):
         mock_file.assert_called_once_with('synth.pkl', 'rb')
         cloudpickle_mock.load.assert_called_once_with(mock_file.return_value)
         assert loaded_instance == synthesizer_mock
+
+    def test__par_error_on_context_columns(self):
```

Comment:
nitpick: our test names usually go `test_{function_name}_{description}` so this one would be like
`test___init___error_sequence_key_in_context` or something like that

Reply:
Fixed:
https://github.com/sdv-dev/SDV/pull/2108/commits/4c11961d057bf6de3e4b6aca6bd2fe8bcbabe173#diff-8f23874079228dbdef42b9e937b8c44ac58a135681b007cabe5d08b187388d69R390